### PR TITLE
Last (old) misc updates (2/4) from v1

### PR DESCRIPTION
**Pull request by jorgeorpinel**
2.

Minor copy edits to use cases

### CODE REVIEW

File: content/docs/use-cases/sharing-data-and-model-files.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ supports Amazon S3, Microsoft Azure Blob Storage, Google Drive, Google Cloud
 Storage, SSH, HDFS, and other remote locations. The list is constantly growing.
 (For a complete list and configuration instructions, refer to `dvc remote add`.)
 
-![](/img/model-sharing-digram.png)
+![](/img/model-sharing-digram.png) _DVC enables independent data storage_
```
Comment (shcheklein) [inline content/docs/use-cases/sharing-data-and-model-files.md:12]: why is it independent? no sure I understand where does it come from to be honest

Comment (jorgeorpinel) [reply content/docs/use-cases/sharing-data-and-model-files.md:12]: Just trying to describe the figure. The datas storage being independent (from code storage) is what enables sharing it easily, no? We can do any other caption, or remove it if you prefer. Doing this mainly for consistency so no strong op.

Comment (shcheklein) [reply content/docs/use-cases/sharing-data-and-model-files.md:12]: yeah, I understand the idea ... it just sounds strange ...  it should be more like `DVC stores data in your storage`, or `Data is stored in your storage, separately from code`

Comment (jorgeorpinel) [reply content/docs/use-cases/sharing-data-and-model-files.md:12]: Done!

**Review by shcheklein — APPROVED** [review]

**Pull request merged**